exceptions.py

```diff
@@ -0,0 +1,12 @@
+"""
+exceptions.py - File that contains all the custom-made exceptions; primarily for http or parsing errors
```

Comment:
In future, find what error is returned when there just isn't any data, or what actually happens, and make a custom error so users know that is what is going and it wasn't some mess-up with the request.